addon-test-support/-private/helpers.js

```diff
@@ -9,6 +9,9 @@ import { getContext as getEmberTestHelpersContext } from './compatibility';
 
 import $ from '-jquery';
 
+let assign = Object.assign;
```

Comment:
This is technically a breaking change, since IE doesn't have `Object.assign`. I believe destructuring assignment is a better way to handle most of the `assign` use cases.

_BTW, you might want to consider v2.0.0-beta where this is solved already._

Reply:
Ahh, I always saw where Object.Assign was suggested. Using the destructure would also be a breaking change though as you would no longer export assign. Technically only meant for internal but someone could have used it. 

Thanks. WIll look at using the beta.2 Try to avoid betas in my app, but since this addon is a dev only concern, might do it.